<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<!-- saved from url=(0052)http://mpf.biol.vt.edu/research/caulobacter/SWST/pp/ -->

<html><head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Spatial Cell Models</title>
<script type="text/javascript" src="./d3/d3.js"> </script>
    <script type="text/javascript" src="./CCData/PleC_population.js"> </script>
    <script type="text/javascript" src="./CCData/PopZ_population.js"> </script>
    <script type="text/javascript" src="./CCData/smallconfine.js"> </script>
    <script type="text/javascript" src="./CCData/smallcontour.js"> </script>
    <script type="text/javascript" src="./CCData/largeconfine.js"> </script>
    <script type="text/javascript" src="./CCData/largecontour.js"> </script>
    <script type="text/javascript" src="./CCData/twoconfine.js"> </script>
    <script type="text/javascript" src="./CCData/twocontour.js"> </script>
    <script type="text/javascript" src="./CCData/midconfine.js"> </script>
    <script type="text/javascript" src="./CCData/midcontour.js"> </script>
    <script type="text/javascript" src="./CCData/lateconfine.js"> </script>
    <script type="text/javascript" src="./CCData/latecontour.js"> </script>
    <script type="text/javascript" src="./CCData/endconfine.js"> </script>
    <script type="text/javascript" src="./CCData/endcontour.js"> </script>
    <link href="./css/style.css" rel="stylesheet" type="text/css"/>
<meta name="Description" content="content=&quot;A" mathematical="" model="" of="" cell="" cycle="" control="" in="" caulobacter"="">
<meta name="keywords" content="mathematical model Caulobacter model cell cycle control Caulobacter crescentus computational biology systems biology modeling biological network CtrA GcrA DnaA Caulobacter mutants">
<meta name="robots" content="ALL">
<meta name="rating" content="General">
<link href="./css/buddingYeast.css" title="main" rel="stylesheet" type="text/css">
<link href="./css/text_only.css" title="Text only" rel="alternate stylesheet" type="text/css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

<script>


</script>
<style type="text/css"></style>

<style type="text/css"></style></head>
<body>
<div id="container">
<div id="top"><h1 style="text-align:center">Spatial Cell Models</h1></div>
<table id="column_table" width="100%" border="0" cellspacing="0" cellpadding="0">
  <tbody><tr>
  <td id="side_column" valign="top">
<div id="navcontainer">
<script language="JavaScript">

var menu_sections=new Array("intro","stats", "wildtype", "mutants2", "modelodes", "tools", "other", "who");

/*
 *  Put sub menus as keys here and use their parents as ids.  This is used later
 *  to prevent opening of sub menus from closing the parent menu.
 */
var menu_parents=new Object();
menu_parents["justification"] = "model";


function hideshow(id) {
    this_box = document.getElementById(id);
    if (this_box.style.display == "none")  {
        this_box.style.display = "";
    } else {
        this_box.style.display = "none";
    }
  
  for (i = 0; i <= (menu_sections.length - 1); i++){
    if (menu_sections[i] != id && menu_sections[i] != menu_parents[id]) {
      document.getElementById(menu_sections[i]).style.display = "none";
    }
  }

} 

function show_diagram(){
diagram = window.open('../pp/clickable_diag.php','diagram','width=794,height=536');
diagram.focus();
}
</script>


<ul id="mainList">
  <li><a href="././index.html">Home</a></li>
  <li><a href="./GenericCellCycle.html">Generic Cell Cycle</a></li>
  <li><a href="#" onclick="hideshow(&#39;CellGrowth&#39;)">Cell Growth Models <i class="fa fa-caret-down" style="font-size: 18px"></i></a></li>
    <ul id="CellGrowth" style="display:none">
    <li><a class="" href="./FrogEggModel.html">Frog Egg Model</a></li>
    <li><a class="" href="./BuddingYeastCell.html">Budding Yeast Cell Model</a></li>
    <li><a class="" href="./FissionYeastCell.html">Fission Yeast Cell Model</a></li>
    </ul>
  <li><a href="#" onclick="hideshow(&#39;Caulobacter&#39;)">Caulobacter Model <i class="fa fa-caret-down" style="font-size: 18px"></i></a></li>
    <ul id="Caulobacter" style="display:none">
    <li><a class="" href="./Caulobacter.html">Caulobacter Introduction</a></li>
    <li><a class="" href="./TemporalModules.html">Temporal Cell Modules</a></li>
    <li><a class="" href="./SpatialModules.html">Spatial Cell Modules</a></li>
    </ul>
  <li><a href="./InterestingFacts.html">Interesting Cell Cycle Facts</a></li>
  <li><a href="./Visualization.html">Visualization</a></li>
  <li><a href="./Contact.html">Contact</a></li>
</ul>


</div>
</td>
  <td id="content_cell" valign="top">
  <div id="content">
<style type="text/css">

.style1 {
  font-size: 12px;
  font-weight: bold;
}
</style>

<div id="PopZ Localization"><p style="color:rgb(16, 16, 142);font-size: 36;"><b>PopZ Localization</b></p></div>
<div id="PopZ Model"><p style="font-size: 30px"><b><u>Model</u></b></p></div>
<div id="Space1"><p style="font-size: 12px">&nbsp;</p></div>
<h2>Physiology of protein localization in the <em>Caulobacter</em> cell cycle</h2>

<p>In <em>Caulobacter crescentus</em>, a model organism for the study of protein localization [1], the polymer PopZ has been identified as a potential landmark protein [2,3]. PopZ localizes to the old pole of newborn cells, and then attains a second zone of polymerization at the new pole in predivisional cells (see Figure 4.1). One function of PopZ is to act as a scaffold tethering segregated chromosomes to the two poles of the cell [4]. As the chromosomes segregate to opposite poles of the cell, the cell division protein FtsZ (a tubulin homologue) polymerizes near the midpoint of the cell, at the future site of cell division. FtsZ polymerization at the correct time and place is a consequence of chromosome segregation. The moving chromosome front carries with it MipZ, a protein that promotes depolymerization of FtsZ [5,6]. As the replicated chromosomes segregate to opposite poles of the cell, the maximum concentration of MipZ localizes to the poles. Hence, the maximum concentration of FtsZ is found at the center of the cell, where MipZ concentration is lowest. Interestingly, the FtsZ peak is very sharp, much sharper than the drop off in MipZ would suggest.
</p>

<p align="center"><img "="" src="./CCGraph/51.png" width="70%"></p>
<p class="style1" align="center"> Figure 4.1: <strong>Schematic representation of the dynamic localization of poteins during the <em>Caulobacter crescentus</em> cell cycle. PopZ polymer present at the old pole tethers the chromosome by docking the <em>parS</em>-ParB centromeric complex. Later in the cell cycle, PopZ polymer appears at the new pole, where it docks the newly replicated <em>parS</em>-ParB front. MipZ, which co-localizes with ParB, is a negative regulator of FtsZ polymerization. Hence, FtsZ, which promotes cell division, is always found to localize at the position in the cell that has the lowest concentration of MipZ.
</strong></p>

<p>
	While the dynamic localization of PopZ is clearly observed and its role in the cell cycle understood, the mechanism behind PopZ localization is still being investigated and debated. Any one or a combination of these mechanisms may be responsible for new-pole localization of PopZ; or it may be that none of them play a definitive role in PopZ localization. In this paper we argue that PopZ localization may depend on a Turing-type mechanism [7,8] by comparing the predictions of a mathematical model to well established properties of PopZ patterns. We also propose that the sharp FtsZ band at the site of cell division is a consequence of a Turing-type mechanism of self-activating assembly and focusing. Our final model of the PopZ-MipZ-FtsZ interaction network will be formulated as a set of partial differential equations and solved using the Method of Lines (Methods). Simulations of the model reproduce experimentally observed spatiotemporal distributions of these proteins during the cell cycle of wild-type and mutant <em>Caulobacter</em> cells.
</p>

<p></p>

<h2>Spatiotemporal dynamics of Turing patterns</h2>

<p>
Turing models are best known for explaining patterns that are periodic in space [7,8]. The theory of Turing patterns predicts that stable, stationary patterns can be generated for a two-species system if two conditions are satisfied: one species must be self-activating and slow diffusing, while the other species must diffuse significantly faster and have an inhibitory effect on the self-activating species. In the Activator-Inhibitor Production (A-IP) version of Turing's mechanism, the initial patch of activator forms by chance, followed by local production of inhibitor molecules, which diffuse rapidly away from the activator patch and inhibit production of activator in the surrounding region [9-11]. At some distance away, the inhibitor concentration is low enough to permit a new zone of activator self-amplified accumulation. An alternative version of the scheme is the Activator-Substrate Depletion (A-SD) mechanism, where a substrate is converted to product by an autocatalytic reaction [10,11]. Local accumulation of the self-activating product depletes surrounding regions of the rapidly diffusing substrate. Only some distance away will substrate concentration be large enough to support a new zone of product accumulation. This critical distance between activator peaks (where inhibitor concentration is low enough or substrate concentration is high enough) is the characteristic wavelength, &lambda;<sub>0</sub>, of a Turing pattern (Figure 4.2).

</p>


<p align="center"><img "="" src="./CCGraph/52.png" width="70%"></p>
<p class="style1" align="center"> Figure 4.2: <strong>Two scenarios for the function of PleC (kinase or phosphatase) in the early predivisional (PD) cell. Spatiotemporal dynamics of PleC (green) and DivL (dark blue) during the cell cycle under these two scenarios.
</strong></p>


<p></p>

<h2>Method</h2>


<p>
We model Turing patterns in one spatial dimension x, by RDEs for activator concentration a(x,t) and substrate concentration s(x,t):</p>

<p align="center"><img "="" src="./CCGraph/4e1.png" width="85%"></p>

<p>
	The "classical Geirer-Meinhardt" equations lack the terms k<sub>dnv</sub>&#8226;s and k<sub>as</sub>&#8226;a, which correspond to de novo synthesis of activator (polymer) from substrate (monomer) and to dissociation of polymer into monomers. We solve the RDEs (1-2) in one spatial dimension, by the method of lines [12]. That is, we discretize the spatial dimension into n compartments (n = 100) of length h = L/n, and approximate it by a central difference scheme (Eq. 3a). We implement no-flux boundary conditions as described in Eq. (3b,c). (In these equations, c<sub>i</sub> represents the concentration of either activator or substrate in compartment i.)
</p>
<p align="center"><img "="" src="./CCGraph/4e2.png" width="85%"></p>

<p>
	The resulting 2n+1 ordinary differential equations are solved using MATLAB's ode15s solver. 
</p>

<div id="PopZ Results"><p style="font-size: 30px"><b><u>Results</u></b></p></div>
<h3>1. Transient polar localization of a nucleating factor may generate bipolar PopZ patterns</h3>

<p align="center"><img "="" src="./CCGraph/53.png" width="60%"></p>
<p class="style1" align="center"> Figure 4.3: <strong>Presence of a nucleating factor at the poles can bias PopZ self-assembly at the poles.
Color indicates concentration gradients from minimum (blue) to maximum (red). (A) Nucleating factor is present uniformly across the length of the cell during the cell cycle. (B) Nucleating factor is present in five compartments at each pole during the course of the cell cycle.
</strong></p>




<h3>2. Bipolar patterns of PopZ can be generated in the absence of nucleating factors</h3>

<p align="center"><img "="" src="./CCGraph/55.png" width="55%"></p>
<p class="style1" align="center"> Figure 4.4: <strong>Unipolar to bipolar transitions in PopZ localization can be realized in the absence of nucleating factors. Color indicates concentration gradients from minimum (blue) to maximum (red). (A) One division cycle (150 min). Left panel: k<sub>dnv</sub> = 1 min<sup>-1</sup>; a single PopZ focus forms at the center of the cell. Right panel: k<sub>dnv</sub> = 12 min<sup>-1</sup>; a PopZ focus forms at the old end of the cell, and a second focus forms at the new end at t = 100 min. (B) A non-dividing (filamentous) cell. Left panel: k<sub>dnv</sub> = 1 min<sup>-1</sup>; PopZ polymers undergo splitting to form a branched pattern as the cell grows. Right panel: k<sub>dnv</sub> = 1 min<sup>-1</sup>; new PopZ foci arise de novo as the cell grows.
</strong></p>

<h3>3. Localization of slow diffusing popZ mRNA suggests that bipolarity requires the presence of two popZ genes</h3>

<p align="center"><img "="" src="./CCGraph/56.png" width="70%"></p>
<p class="style1" align="center"> Figure 4.5: <strong>Stochastic simulations show that limited dispersal of mRNA produced from a single gene results in unipolar PopZ patterns. One popZ gene is present throughout the cell cycle at a distance of 0.2&#8226;Lt from the old pole producing mRNA with a diffusion coefficient of 0.05 &mu;m<sup>2</sup>min<sup>-1</sup>. For popZ gene, color indicates presence (blue) or absence (yellow) of the gene. For popZ mRNA and PopZ protein, color indicates concentration gradients from minimum (blue) to maximum (red).
</strong></p>

<p align="center"><img "="" src="./CCGraph/58.png" width="70%"></p>
<p class="style1" align="center"> Figure 4.6: <strong>Stochastic simulation of the two-gene model. (A) One popZ gene (red) is constantly present at a distance of 0.2&#8226;Lt from the old pole, while a second popZ gene (blue) is introduced at t = 50 min. The second popZ gene is translocated across the cell until it reaches a distance of 0.2&#8226;Lt from the new pole. popZ mRNA (D<sub>mRNA</sub> = 0.05 &mu;m<sup>2</sup>min<sup>-1</sup>) is produced by both genes. PopZ polymer shows a unipolar-to-bipolar transition at t = 92 min. For popZ gene, color indicates presence (blue) or absence (yellow) of the gene. For popZ mRNA and PopZ protein, color indicates concentration gradients from minimum (blue) to maximum (red). (B) Distribution of the time (left panel) and cell size (right panel) at which popZ gene segregation is completed (green bars) and PopZ becomes bipolar (red bars).
</strong></p>

<h3>4. The two-gene model of PopZ localization recreates distribution patterns observed in mutant cells</h3>

<p align="center"><img "="" src="./CCGraph/59.png" width="70%"></p>
<p class="style1" align="center"> Figure 4.7: <strong>The two-gene model can also account for observed PopZ distribution patterns in mutant cells. For popZ gene, color indicates presence (blue) or absence (yellow) of the gene. For popZ mRNA and PopZ protein, color indicates concentration gradients from minimum (blue) to maximum (red). (A) Shifting the position of a single popZ gene from the old pole to the new pole results in a corresponding shift in the polar localization of PopZ polymer (k<sub>syn-mRNA</sub> = 2 min<sup>-1</sup>, D<sub>mRNA</sub> = 0.05 &mu;m<sup>2</sup>min<sup>-1</sup>). (B) Transient bipolar localization of PopZ is observed under mild-overproduction (ksyn-mRNA = 3 min−1) and faster diffusion of mRNA (D<sub>mRNA</sub> = 0.5 &mu;m<sup>2</sup>min<sup>-1</sup>). (C) Overexpression (k<sub>syn-mRNA</sub> = 10 min<sup>-1</sup>) results in PopZ polymerizing further into the cytoplasm.
</strong></p>

<div id="Buffer Space"><p style="font-size: 12px">&nbsp;</p></div>
<div id="Chromosome Segregation"><p style="color:rgb(16, 16, 142);font-size: 36;"><b>Chromosome Segregation</b></p></div>
<div id="Chromosome Segregation Model"><p style="font-size: 30px"><b><u>Model</u></b></p></div>
<div id="Space4"><p style="font-size: 12px">&nbsp;</p></div>

<h2>Spatiotemporal Model of Chromosome Segregation in <em>Caulobacter crescentus</em></h2>

<p>In recent years, ParABS-mediated chromosomal segregation has been frequently studied in the bacterium <em>Caulobacter crescentus</em> [1-6]. The <em>parS</em> locus is located 8 kb away from the origin of replication [7], which in turn is oriented towards the old (stalked) pole of the <em>Caulobacter</em> cell. The ParB protein binds specifically to the <em>parS</em> sequence forming a centromere-like complex. Chromosome replication results in two copies of the <em>parS</em>/ParB complex. While the old copy is retained at the old pole, the new centromeric complex or 'chromosome front' is translocated to the opposite pole in a matter of minutes [1,4,6]. Fluorescence microscopy experiments demonstrate that the <em>parS</em>/ParB complex follows a retracting cloud of ParA towards the new pole [1,2], suggesting that ParA participates in the segregation of the replicated bacterial chromosome (Figure 5.1).

</p>

<p align="center"><img "="" src="./CCGraph/61.png" width="70%"></p>
<p class="style1" align="center"> Figure 5.1: <strong>Schematic illustration of the consensus model of chromosome segregation in <em>Caulobacter crescentus</em>. In the swarmer cell, a ParA concentration gradient (pink) extends from a maximum at the new pole to a minimum at the old pole. The <em>parS</em>/ParB chromosome front (green circles) is tethered to the single focus of PopZ (red). Once DNA replication is initiated in the stalked cell, a second copy of the <em>parS</em>/ParB chromosome front translocates behind the receding ParA structure. Concurrent with the completion of segregation in the early predivisional cell, a new focus of PopZ is formed at the new pole to tether the newly replicated chromosome front. TipN (blue circle) is localized at the new pole in the early stages of the cell cycle before being repositioned at mid-cell in the late predivisional stage.
</strong></p>

<p></p>

<h2>Method</h2>


<p>
<strong>Spatial discretization.</strong> The proteins under investigation in this work are distributed heterogeneously along the long axis of a <em>Caulobacter</em> cell. Hence, it is sufficient to consider biochemical reactions and diffusion occurring in one spatial dimension, and to write the governing partial differential equation (PDE) for a generic chemical species C as:
</p>

<p align="center"><img "="" src="./CCGraph/3e1.png" width="35%"></p>

<p><strong><em>parS</em>/ParB segregation.</strong> All variables in our differential equation model have units of concentration. There can be only one or two <em>parS</em> sequences at any given stage of the cell cycle; hence, defining <em>parS</em> in terms of a real number variable S(x,t) is not accessible to our model. Instead, we treat the chromosome sequence <em>parS</em> and the protein ParB as a single polymeric entity. Furthermore, for microscopy experiments reported in literature, ParB is tagged and used as a proxy for <em>parS</em> movement in the cell. Hence, representing the two as single entity reflects the protocol of the experimental observations we are trying to replicate in simulation.
</p>

<p align="center"><img "="" src="./CCGraph/6e1.png" width="80%"></p>

<p><strong>ParA biochemical pathway.</strong> Since we desire to investigate <em>parS</em> segregation in both ParA wild-type and mutant backgrounds, a description of ParA's interactions with ParB, DNA and other ParA molecules is important. Figure 5.2 shows the biochemical reactions that form the bases for the equations in our model. The numbers of ParA molecules present in the cell are not sufficient to make up a polymer filament that is long enough to span the long axis of the <em>Caulobacter</em> cell, making it unlikely for the chromosome to be segregated by a ParA filament [4]. Hence, we do not include the ParA polymer species in or model. We desired to simulate three important ParA mutants in our model, namely (i) ParA<sub>G16V</sub>, which is a dimerization-negative mutant, (ii) ParA<sub>R195E</sub>, which is deficient in DNA binding, and (iii) ParAD44A, in which the ATPase activity is blocked. For each of these mutants, the rate constant for the corresponding reaction is set to zero.
</p>

<p align="center"><img "="" src="./CCGraph/62.png" width="70%"></p>
<p class="style1" align="center"> Figure 5.2: <strong>Proposed ParA biochemical cycle ParA monomers (pink) can bind TipN (blue), PopZ (red), or form dimers. ParA dimers bind non- specifically to DNA. The <em>parS</em>/ParB chromosome front (green) has high affinity for DNA-bound ParA dimers. ParB up-regulates the ATPase activity of ParA, resulting in the release of ParA in the form of free monomers.
</strong></p>

<p>
	FtsZ, TipN and MipZ localization dynamics. In our model, MipZ is present in a freely diffusing monomeric form and a <em>parS</em>-bound form that cannot diffuse. FtsZ is represented in both monomeric and polymeric forms. Polymer formation occurs by de novo and autocatalytic growth. The rate of FtsZ depolymerization is a function of total MipZ. The localization of TipN depends indirectly on FtsZ [8]. For simplicity however, we assume TipN directly binds to and co- localizes with FtsZ.
</p>

<p>
	PopZ. In vivo, the PopZ protein assembles into polymeric aggregates that localize at the old pole in swarmer cells and later assumes bipolar distribution [9,10]. New pole localization of PopZ requires DNA replication [11] and chromosome segregation [12]. In earlier work, we formulated a model to show that PopZ localization pattern can be explained based on a Turing mechanism. Further, we proposed that slow diffusion of popZ mRNA makes it necessary for the popZ gene to be present close to both poles for bipolar PopZ localization. The current model does not include popZ genes or mRNA. In this preliminary model, we set the rate of PopZ monomer production to be dependent on the concentration of the <em>parS</em>/ParB focus. As a result, the emergence of a PopZ focus at the new pole coincides with the completion of chromosome segregation.
</p>

<div id="Buffer Space"><p style="font-size: 12px">&nbsp;</p></div>
<div id="Chromosome Segregation Results"><p style="font-size: 30px"><b><u>Results</u></b></p></div>
<h3>1. The model is able to reproduce the wild type distribution of ParA, <em>parS</em>/ParB, PopZ and FtsZ.</h3>

<p align="center"><img "="" src="./CCGraph/63.png" width="60%"></p>
<p class="style1" align="center"> Figure 5.3: <strong>Spatiotemporal distribution of ParA (right panel) and <em>parS</em>/ParB chromosome fronts in wild-type and mutant cells. Colors indicate concentration gradients from minimum (blue) to maximum (red). (A) Wild type, (B) &Delta;tipN, and (C) ParA overexpression. (D) parA<sub>D44A</sub> mutants express ParA<sub>D44A</sub> proteins that cannot be hydrolyzed by ParB. (E) parA<sub>G16V</sub> mutants express ParA<sub>G16V</sub> proteins that cannot form dimers. (F) parA<sub>R195E</sub> mutants express ParA<sub>R195E</sub> proteins that cannot bind DNA.
</strong></p>

<h3>2. The model reproduces delays in chromosome segregation and FtsZ mid-cell localization in &Delta;tipN and ParA overexpression background</h3>

<p align="center"><img "="" src="./CCGraph/64.png" width="60%"></p>
<p class="style1" align="center"> Figure 5.4: <strong>Spatiotemporal distribution of FtsZ (left panel) and PopZ (right panel) in wild-type and mutant cells. Colors indicate concentration gradients from minimum (blue) to maximum (red). (A) Wild type, (B) &Delta;tipN, and (C) ParA overexpression. (D) parA<sub>D44A</sub> mutants express ParA<sub>D44A</sub> proteins that cannot be hydrolyzed by ParB. (E) parA<sub>G16V</sub> mutants express ParA<sub>G16V</sub> proteins that cannot form dimers. (F) parA<sub>R195E</sub> mutants express ParAR195E proteins that cannot bind DNA.
</strong></p>

<h3>3. The model reproduces aberrant localization of ParA mutants. PopZ may regulate chromosome segregation via binding ParA dimers instead of ParA monomers.</h3>

<p align="center"><img "="" src="./CCGraph/65.png" width="80%"></p>
<p class="style1" align="center"> Figure 5.5: <strong>Centromere segregation requires PopZ to interact with the dimeric form of ParA. Colors indicate concentration gradients from minimum (blue) to maximum (red). The time taken for translocating the <em>parS</em>/ParB chromosome front (upper panel), and the ParA distribution profile (lower panel) are comparable between simulations of the (A) wild type strain, and (B) a mutant strain containing a PopZ variant that cannot bind ParA monomer (popZ-SP1). (C) Simulations of a strain containing a PopZ variant that cannot bind ParA dimers (popZ-SP2) shows that the <em>parS</em>/ParB segregation is delayed.
</strong></p>




<h3><em>Reference</em></h3>
<ol>
	<li>PopZ Localization References</li>
    <ol>
        <li>Goley ED, Iniesta AA, Shapiro L (2007) Cell cycle regulation in Caulobacter: location, location, location. J Cell Sci 120: 3501-3507.</li>
        <li>Ebersbach G, Briegel A, Jensen GJ, Jacobs-Wagner C (2008) A self-associating protein critical for chromosome attachment, division, and polar organization in caulobacter. Cell 134: 956-968.</li>
        <li>Bowman GR, Comolli LR, Zhu J, Eckart M, Koenig M, et al. (2008) A polymeric protein anchors the chromosomal origin/ParB complex at a bacterial cell pole. Cell 134: 945-955.</li>
        <li>Bowman GR, Comolli LR, Gaietta GM, Fero M, Hong S-H, et al. (2010) Caulobacter PopZ forms a polar subdomain dictating sequential changes in pole composition and function. Mol Microbiol 76: 173-189.</li>
        <li>Thanbichler M, Shapiro L (2006) MipZ, a spatial regulator coordinating chromosome segregation with cell division in Caulobacter. Cell 126: 147-162.</li>
        <li>Kiekebusch D, Michie KA, Essen LO, Lowe J, Thanbichler M (2012) Localized Dimerization and Nucleoid Binding Drive Gradient Formation by the Bacterial Cell Division Inhibitor MipZ. Mol Cell 46: 245-259.</li>
        <li>Turing AM (1952) The Chemical Basis of Morphogenesis. Philos Trans R Soc B Biol Sci 237: 37-72.</li>
        <li>Kondo S, Miura T (2010) Reaction-diffusion model as a framework for understanding biological pattern formation. Science 329: 1616-1620.</li>
        <li>Gierer A, Meinhardt H (1972) A theory of biological pattern formation. Kybernetik 12: 30-39.</li>
        <li>Meinhardt H, Gierer A (2000) Pattern formation by local self-activation and lateral inhibition. Bioessays 22: 753-760.</li>
        <li>Meinhardt H (1982) Models of biological pattern formation. Research Gate. pp. 1-10.</li>
        <li>Schiesser WE (1991) The Numerical method of lines. Academic Press.</li>
    </ol>
	<li>Chromosome Segregation References</li>
    <ol>
        <li>Schofield WB, Lim HC, Jacobs-Wagner C (2010) Cell cycle coordination and regulation of bacterial chromosome segregation dynamics by polarly localized proteins. Eur Mol Biol Organ J 29: 3068-3081.</li>
        <li>Ptacin JL, Lee SF, Garner EC, Toro E, Eckart M, et al. (2010) A spindle-like apparatus guides bacterial chromosome segregation. Nat Cell Biol 12: 791-798.</li>
        <li>Toro E, Hong S-H, McAdams HH, Shapiro L (2008) Caulobacter requires a dedicated mechanism to initiate chromosome segregation. Proc Natl Acad Sci U S A 105: 15435-15440.</li>
        <li>Lim HC, Surovtsev VI, Beltran BG, HUang F, Bewesdorf Jorg, et al. (2014) Evidence for a DNA-relay mechanism in ParABS-mediated chromosome segregation. Elife 3.</li>
        <li>Ptacin JL, Gahlmann A, Bowman GR, Perez AM, von Diezmann ARS, et al. (2014) Bacterial scaffold directs pole-specific centromere segregation. Proc Natl Acad Sci U S A.</li>
        <li>Shebelut CW, Guberman JM, Van Teeffelen S, Yakhnina AA, Gitai Z (2010) Caulobacter chromosome segregation is an ordered multistep process. Proc Natl Acad Sci U S A 107: 14194-14198.</li>
        <li>Hong S-H, McAdams HH (2011) Compaction and transport properties of newly replicated Caulobacter crescentus DNA. Mol Microbiol 82: 1349-1358.</li>
        <li>Yeh Y-C, Comolli LR, Downing KH, Shapiro L, McAdams HH (2010) The Caulobacter Tol- Pal complex is essential for outer membrane integrity and the positioning of a polar localization factor. J Bacteriol 192: 4847-4858.</li>
        <li>Ebersbach G, Briegel A, Jensen GJ, Jacobs-Wagner C (2008) A self-associating protein critical for chromosome attachment, division, and polar organization in Caulobacter. Cell 134: 956-968.</li>
        <li>Bowman GR, Comolli LR, Zhu J, Eckart M, Koenig M, et al. (2008) A polymeric protein anchors the chromosomal origin/ParB complex at a bacterial cell pole. Cell 134: 945-955.</li>
        <li>Bowman GR, Comolli LR, Gaietta GM, Fero M, Hong S-H, et al. (2010) Caulobacter PopZ forms a polar subdomain dictating sequential changes in pole composition and function. Mol Microbiol 76: 173-189.</li>
        <li>Laloux G, Jacobs-Wagner C (2013) Spatiotemporal control of PopZ localization through cell cycle-coupled multimerization. J Cell Biol 201: 827-841.</li>
    </ol>
</ol>



<!-- <p>The movies below show the population of PopZ, mRNA and Gene changing during the division process.</p>

<center>

<P>
<EMBED SRC="MultiCurve.avi" autostart="false" width="680" height="500">
</P>

<p class="style1"> Movie 1. PopZ population of four different simulations. If your browser is unable to play this video, right-click over <a href="http://bioinformatics.cs.vt.edu/~cmhshirl/MultiCurve.avi.zip"><em>the link</em></a> and choose save target as.
</p>

<P>
<EMBED SRC="meanCurve.avi" autostart="false" width="680" height="500">
</P>

<p class="style1"> Movie 2. Average population of PopZ, mRNA and Gene (100 simulations). If your browser is unable to play this video, right-click over <a href="http://bioinformatics.cs.vt.edu/~cmhshirl/meanCurve.avi.zip"><em>the link</em></a> and choose save target as.
</p>
 -->


</div> <!-- end #content -->
</td></tr>
</tbody></table>


<div id="FOOTER">
<span style="float:right">Page designed April 26, 2022</span><span style="float:left">Site Design by <a class="credits" style="font-weight:800">  © Tyson's Lab</a></span><center><a class="credits" style="font-weight:800" href="mailto:cmhshirl@vt.edu"><img border="0" src="./CCGraph/mailto.gif"> webmaster</a></center>
<div style="clear:both"></div>
</div>
</div>

<script src="drawPleC.js"></script>
<script src="drawPopZ.js"></script>

<script type="text/javascript">


  document.getElementById('project').addEventListener('change', function(){
      if(document.getElementById('project').value =="none"){
        document.getElementById("PleCM").hidden = "hidden";
        document.getElementById("PopZM").hidden = "hidden";
        document.getElementById("g-play-button").hidden = "hidden";
        d3.selectAll("svg").remove();
       }
      else if(document.getElementById('project').value =="PleC"){
        document.getElementById("PleCM").hidden = "";
        document.getElementById("PopZM").hidden = "hidden";
        document.getElementById("g-play-button").hidden = "hidden";
        document.getElementById('project').disabled = true 
      }
      else if(document.getElementById('project').value =="PopZ"){
        document.getElementById("PleCM").hidden = "hidden";
        document.getElementById("PopZM").hidden = "";
        document.getElementById("g-play-button").hidden = "hidden";
        document.getElementById('project').disabled = true      
      }
   });

  function visPleC(){
      var speFlag=[0,0,0,0,0,0,0,0,0,0,0];
      var speNum = 11;
      var species = ["0","1","2","3","4","5","6","7","8","9","10"];
      var label = ["000","101","102","103","104","105","106","107","108","109","110"];

      var j=0;
      var legend = ["none","red","green","blue","cyan","magenta","orange","maroon","lime","black","yellow"];
      for(var i=1; i<speNum; i++){
        if(document.getElementById(species[i]).checked){
          speFlag[i]=1;
          j=j+1
          document.getElementById(label[i]).style.color = legend[j]
          document.getElementById(label[i]).style.fontWeight = "bold"
        }
        else {
          speFlag[i]=0;
          document.getElementById(label[i]).style.color = "grey"
        } 
      document.getElementById(species[i]).disabled = true 
      document.getElementById('submitPleC').disabled = true
      }
      showPleC(speFlag); 
      document.getElementById("g-play-button").hidden = "";
      beginPleC()
  } 

  function visPopZ(){
      var speFlag=[0,0,0,0,0];
      var speNum = 5;
      var species = ["0","21","22","23","24"];
      var label = ["000","201","202","203","204"];

      var j=0;
      var legend = ["none","red","green","blue","cyan","magenta","orange","maroon","lime","black","yellow"];
      for(var i=1; i<speNum; i++){
        if(document.getElementById(species[i]).checked){
          speFlag[i]=1;
          j=j+1
          document.getElementById(label[i]).style.color = legend[j]
          document.getElementById(label[i]).style.fontWeight = "bold"
        }
        else {
          speFlag[i]=0;
          document.getElementById(label[i]).style.color = "grey"
        } 
      document.getElementById(species[i]).disabled = true 
      document.getElementById('submitPopZ').disabled = true
      }
      showPopZ(speFlag); 
      document.getElementById("g-play-button").hidden = "";
      beginPopZ()
  }
</script>




<script type="text/javascript">

/***********************************************
* http://www.dynamicdrive.com/dynamicindex5/dhtmltooltip.htm
***********************************************/

var offsetxpoint=-60 //Customize x offset of tooltip
var offsetypoint=20 //Customize y offset of tooltip
var ie=document.all
var ns6=document.getElementById && !document.all
var enabletip=false
if (ie||ns6)
var tipobj=document.all? document.all["dhtmltooltip"] : document.getElementById? document.getElementById("dhtmltooltip") : ""

function ietruebody(){
return (document.compatMode && document.compatMode!="BackCompat")? document.documentElement : document.body
}

function ddrivetip(thetext, thecolor, thewidth){
if (ns6||ie){
if (typeof thewidth!="undefined") tipobj.style.width=thewidth+"px"
if (typeof thecolor!="undefined" && thecolor!="") tipobj.style.backgroundColor=thecolor
tipobj.innerHTML=thetext
enabletip=true
return false
}
}

function positiontip(e){
if (enabletip){
var curX=(ns6)?e.pageX : event.x+ietruebody().scrollLeft;
var curY=(ns6)?e.pageY : event.y+ietruebody().scrollTop;
//Find out how close the mouse is to the corner of the window
var rightedge=ie&&!window.opera? ietruebody().clientWidth-event.clientX-offsetxpoint : window.innerWidth-e.clientX-offsetxpoint-20
var bottomedge=ie&&!window.opera? ietruebody().clientHeight-event.clientY-offsetypoint : window.innerHeight-e.clientY-offsetypoint-20

var leftedge=(offsetxpoint<0)? offsetxpoint*(-1) : -1000

//if the horizontal distance isn't enough to accomodate the width of the context menu
if (rightedge<tipobj.offsetWidth)
//move the horizontal position of the menu to the left by it's width
tipobj.style.left=ie? ietruebody().scrollLeft+event.clientX-tipobj.offsetWidth+"px" : window.pageXOffset+e.clientX-tipobj.offsetWidth+"px"
else if (curX<leftedge)
tipobj.style.left="5px"
else
//position the horizontal position of the menu where the mouse is positioned
tipobj.style.left=curX+offsetxpoint+"px"

//same concept with the vertical position
if (bottomedge<tipobj.offsetHeight)
tipobj.style.top=ie? ietruebody().scrollTop+event.clientY-tipobj.offsetHeight-offsetypoint+"px" : window.pageYOffset+e.clientY-tipobj.offsetHeight-offsetypoint+"px"
else
tipobj.style.top=curY+offsetypoint+"px"
tipobj.style.visibility="visible"
}
}

function hideddrivetip(){
if (ns6||ie){
enabletip=false
tipobj.style.visibility="hidden"
tipobj.style.left="-1000px"
tipobj.style.backgroundColor=''
tipobj.style.width=''
}
}

document.onmousemove=positiontip

</script>


</body></html>

