<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<!-- saved from url=(0052)http://mpf.biol.vt.edu/research/caulobacter/SWST/pp/ -->

<html><head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Discover the Cell Cycle</title>
<script type="text/javascript" src="./d3/d3.js"> </script>
    <script type="text/javascript" src="./CCData/PleC_population.js"> </script>
    <script type="text/javascript" src="./CCData/PopZ_population.js"> </script>
    <script type="text/javascript" src="./CCData/smallconfine.js"> </script>
    <script type="text/javascript" src="./CCData/smallcontour.js"> </script>
    <script type="text/javascript" src="./CCData/largeconfine.js"> </script>
    <script type="text/javascript" src="./CCData/largecontour.js"> </script>
    <script type="text/javascript" src="./CCData/twoconfine.js"> </script>
    <script type="text/javascript" src="./CCData/twocontour.js"> </script>
    <script type="text/javascript" src="./CCData/midconfine.js"> </script>
    <script type="text/javascript" src="./CCData/midcontour.js"> </script>
    <script type="text/javascript" src="./CCData/lateconfine.js"> </script>
    <script type="text/javascript" src="./CCData/latecontour.js"> </script>
    <script type="text/javascript" src="./CCData/endconfine.js"> </script>
    <script type="text/javascript" src="./CCData/endcontour.js"> </script>
    <link href="./css/style.css" rel="stylesheet" type="text/css"/>
<meta name="Description" content="content=&quot;A" mathematical="" model="" of="" cell="" cycle="" control="" in="" caulobacter"="">
<meta name="keywords" content="mathematical model Caulobacter model cell cycle control Caulobacter crescentus computational biology systems biology modeling biological network CtrA GcrA DnaA Caulobacter mutants">
<meta name="robots" content="ALL">
<meta name="rating" content="General">
<link href="./css/buddingYeast.css" title="main" rel="stylesheet" type="text/css">
<link href="./css/text_only.css" title="Text only" rel="alternate stylesheet" type="text/css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

<script>


</script>
<style type="text/css"></style>

<style type="text/css"></style></head>
<body>
<div id="container">
<div id="top"><h1 style="text-align:center">Discovering the Cell Cycle</h1></div>
<table id="column_table" width="100%" border="0" cellspacing="0" cellpadding="0">
  <tbody><tr>
  <td id="side_column" valign="top">
<div id="navcontainer">
<script language="JavaScript">

var menu_sections=new Array("intro","stats", "wildtype", "mutants2", "modelodes", "tools", "other", "who");

/*
 *  Put sub menus as keys here and use their parents as ids.  This is used later
 *  to prevent opening of sub menus from closing the parent menu.
 */
var menu_parents=new Object();
menu_parents["justification"] = "model";


function hideshow(id) {
    this_box = document.getElementById(id);
    if (this_box.style.display == "none")  {
        this_box.style.display = "";
    } else {
        this_box.style.display = "none";
    }
  
  for (i = 0; i <= (menu_sections.length - 1); i++){
    if (menu_sections[i] != id && menu_sections[i] != menu_parents[id]) {
      document.getElementById(menu_sections[i]).style.display = "none";
    }
  }

} 

function show_diagram(){
diagram = window.open('../pp/clickable_diag.php','diagram','width=794,height=536');
diagram.focus();
}
</script>


<ul id="mainList">
<li><a href="././index.html">Home</a></li>
<li><a href="./Intro.html">Introduction</a></li>
<li><a href="#" onclick="hideshow(&#39;CellGrowth&#39;)">Cell Growth Models <i class="fa fa-caret-down"></i></a></li>
  <ul id="CellGrowth" style="display:none">
  <li><a class="" href="./FrogEggModel.html">Frog Egg Model</a></li>
  <li><a class="" href="./BuddingYeastCell.html">Budding Yeast Cell Model</a></li>
  </ul>
  <li><a href="./GenericCellCycle.html">Generic Cell Cycle</a></li>
<li><a href="#" onclick="hideshow(&#39;Caulobacter&#39;)">Caulobacter Cell <i class="fa fa-caret-down"></i></a></li>
  <ul id="Caulobacter" style="display:none">
  <li><a class="" href="./FrogEggModel.html">Caulobacter Introduction</a></li>
  <li><a class="" href="./TemporalModels.html">Temporal Cell Models</a></li>
  <li><a class="" href="./SpatialModels.html">Spatial Cell Models</a></li>
  </ul>
<li><a href="#" onclick="hideshow(&#39;PleC&#39;)">Bistable Histidine Kinase</a></li>
  <ul id="PleC" style="display:none">
  <li><a class="" href="./PDM.html">Model</a></li>
  <li><a class="" href="./PDR.html">Results</a></li>
  </ul>
<li><a href="#" onclick="hideshow(&#39;Loc&#39;)">PleC and DivL Localization</a></li>
  <ul id="Loc" style="display:none">
  <li><a class="" href="./DM.html">Model</a></li>
  <li><a class="" href="./DR.html">Results</a></li>
  </ul>
<li><a href="#" onclick="hideshow(&#39;Turing&#39;)">PopZ Localization</a></li>
  <ul id="Turing" style="display:none">
    <li><a class="" id="WT_STSW" style="" href="./TM.html">Model</a></li> 
  <li><a class="" id="WT_Swarmer" style="" href="./TR.html">Results</a></li>    
  </ul>
  
<li><a href="#" onclick="hideshow(&#39;Chromosome&#39;)">Chromosome Segregation</a></li>
  <ul id="Chromosome" style="display:none">
  <li><a id="mutant_menu_ctrA" class="" href="./CM.html">Model</a></li>
<li><a id="mutant_menu_gcrA" class="" href="./CR.html">Results</a></li>
  </ul>
<li><a href="./InterestingFacts.html">Interesting Cell Cycle Facts</a></li>
<li><a href="./Visualization.html">Visualization</a></li>
<li><a href="./Contact.html">Contact</a></li>

</ul>


</div>
</td>
  <td id="content_cell" valign="top">
  <div id="content">
<style type="text/css">

.style1 {
  font-size: 12px;
  font-weight: bold;
}
</style>

<div id="Bistable Histidine"><p style="color:rgb(16, 16, 142);font-size: 36;"><b>Bistable Histidine Kinase</b></p></div>
<div id="Bistable Histidine Model"><p style="font-size: 30px"><b><u>Model</u></b></p></div>
<div id="Space1"><p style="font-size: 12px">&nbsp;</p></div>
<div id="Bistable Histidine"><h2>Bistable histidine kinase switch in <em>Caulobacter crescentus</em></h2></div>

<p>Orchestration of this asymmetric cell division cycle requires proper temporal and spatial regulation of several key proteins (see Figure 2.1 A). The temporal dynamics of these proteins was captured in a pair of papers by Li et al. [1,2]. At least two of these proteins, PleC and CckA, are bifunctional, capable of acting as either phosphatase or kinase. PleC kinase activity is up- regulated by its own response regulator, DivK. It is unknown how DivK alters the activity of its own phosphorylating enzyme, PleC. DivK is present at roughly constant level throughout the cell cycle [3]. However, PleC is a phosphatase during the swarmer stage of the cell cycle and kinase during the stalked stage (see Figure 2.1 B).
</p>

<p align="center"><img "="" src="./CCGraph/21.png" width="55%"></p>
<p class="style1" align="center"> Figure 2.1: <strong>Morphological transitions in <em>Caulobacter crescentus</em> are governed by changes in localization and activity of proteins. (A) Schematic representation of the <em>Caulobacter crescentus</em> cell cycle. The cell undergoes a series of morphological changes from swarmer cell (left)&rarr;nascent stalked cell&rarr;stalked cell&rarr;pre-divisonal cell (right). These events are driven by changes in the activity and localization of cell cycle proteins. In particular, notice that DivL (light blue) and CckA (green) are uniformly distributed on the membrane in the swarmer cell but localized at the poles in the stalked cell. (B) Model of the status of PleC and CckA activity at the old pole in the swarmer cell and in the nascent stalked cell. In the swarmer cell, DivJ is not localized or activated. As a result, PleC is a phosphatase and CckA is a kinase. In the stalked cell, DivJ is localized to the old pole, causing PleC to flip to the kinase form, which in turn induces CckA to switch to a phosphatase.
</strong></p>

<p>
The phosphorylation states of DivK and CtrA are governed by the bifunctional histidine kinases PleC and CckA, respectively. Both PleC and CckA can switch between two conformations: a kinase conformation and a phosphatase conformation [4,5] (see Figure 2.1 B). Typically, in bacteria the change in activity of a bifunctional histidine kinase is brought about by an external signal molecule binding to the sensor region of the protein [6]. However, the change in PleC from a phosphatase to a kinase is brought about by its substrate, DivK [4]. In fact, the sensor domain of PleC is not essential
for its function [7]. This interaction, where substrate binding to a bifunctional histidine kinase changes its function, has, to our knowledge, been observed only for PleC in <em>Caulobacter</em>. It has been suggested that DivK up-regulates PleC kinase activity preferentially in stalked cells because it is in stalked cells where DivK~P and PleC are co-localized at the poles [4].
</p>



<p align="center"><img "="" src="./CCGraph/22.png" width="80%"></p>
<p class="style1" align="center"> Figure 2.2: <strong>The DivJ-PleC-DivK and DivL-CckA-CtrA modules are coupled via DivK~P-DivL interaction. PleC kinase and DivK~P are involved in a positive feedback loop. By phosphorylating DivK, DivJ biases the positive feedback loop toward the PleC kinase state. DivL up-regulates the kinase form of CckA, which in turn phosphorylates CtrA. The phosphorylated form of DivK~P binds to and inactivates DivL, causing CckA to revert to the phosphatase form and dephosphorylate CtrA.
</strong></p>


<p>Taken together, these observations suggest that PleC-DivJ-DivK and DivL-CckA-CtrA are crucial drivers of the swarmer-to-stalked transition, as summarized in Figure 2.1 and Figure 2.2.</p>


<p></p>

<h2>Method</h2>


<p>
Here, we propose a mechanism for ligand-dependent modifications of the bifunctional histidine kinase, PleC. The mechanism consists of elementary chemical reactions describing ligands (either DivK or DivK~P) binding to the histidine kinase dimer in either its phosphatase or kinase form. The binding states determine the rates of the autophosphorylation, phosphotransfer, and phosphatase reactions catalyzed by PleC. If DivK~P is more efficient than unphosphorylated DivK at promoting the transition of PleC from phosphatase to kinase, then PleC and DivK~P would be involved in a positive feedback loop. Such positive feedback loops are well known for their tendency to function as bistable toggle switches [8], and toggle switches are well known for their roles in cellular decision- making [9-11] including critical transitions in the eukaryotic cell cycle [12-14].
</p>

<p>
We have a detailed model of the interactions between DivK and PleC, under reasonable conditions on the rate constants (or propensities) of these reactions, exhibits robust bistability as a function of DivJ activity. The complete reaction network (Figure 2.3) was translated into a system of 52 non-linear ordinary differential equations using the mass-action law of chemical kinetics, with one exception. The mechanism by which DivL promotes the kinase form of CckA is unknown, so we modeled this step phenomenologically with a Hill function. Because there are many closed loops of elementary chemical reactions in Figure 2.3, we must choose rate constant values that respect the thermodynamic principle of detailed balance. As long as we satisfy these thermodynamic constraints, we find that the reaction network exhibits bistability over a robust range of parameter values. 
</p>

<p align="center"><img "="" src="./CCGraph/23A.png" width="80%"></p>
<p class="style1" align="center"> Figure 2.3: <strong>Chemical reaction networks on which the model is based. Here we only show the PleC-DivK system.
</strong></p>

<p>
The equations for the full model were encoded as .ode files and simulated using the freely available software, XPP-AUT. The signal-response curves were drawn using the AUTO facility of XPP-AUT. From the data points generated by XPP-AUT, the plots shown in the figures were generated using the python library, Matplotlib [15].
</p>
&nbsp;

<div id="Bistable Histidine Results"><p style="font-size: 30px"><b><u>Results</u></b></p></div>

<h3>1. DivJ-dependent phosphorylation of DivK is crucial for switching PleC from a phosphatase to a kinase</h3>


<p align="center"><img "="" src="./CCGraph/23.png" width="60%"></p>
<p class="style1" align="center"> Figure 2.4: <strong>DivJ initiates the PleC phosphatase-to-kinase transition. For the full model, we plot signal-response curves (one-parameter bifurcation diagrams) for the steady state levels of (A) PleC kinase, (B) DivK~P, (C) PleD~P, and (D) DivK bound to PleC as functions of total DivJ (the parameter DivJ_tot in the model). Solid lines, stable steady states; dashed lines, unstable steady states. In our model, PleC_tot = constant = 1.0, but total DivK, total CtrA, and total PleD depends on DivJ_tot.
</strong></p>


<h3>2. DivJ-dependent phosphorylation of DivK is crucial for switching PleC from a phosphatase to a kinase</h3>


<p align="center"><img "="" src="./CCGraph/24.png" width="75%"></p>
<p class="style1" align="center"> Figure 2.5: <strong>Over-expressing DivK causes a drop in PleD phosphorylation.
The steady state level of PleD~P is plotted against increasing amount of total DivK for (A) &Delta;<em>divJ divK</em><sub>D53N</sub>, (B) &Delta;<em>divJ</em>, and (C) wild type background. Although the absolute levels vary among the three cell types, in each case PleD~P level shows an initial increase followed by a drop at high DivK.
</strong></p>

<p align="center"><img "="" src="./CCGraph/25.png" width="60%"></p>
<p class="style1" align="center"> Figure 2.6: <strong>Over-expressing DivK causes activation the PleC switch independent of DivJ. (A) Two-parameter bifurcation diagram, indicating how the PleC switch behaves in cells expressing different levels of DivJ_tot and DivK_total (<em>k</em><sub>syn_dk</sub> is the rate constant for synthesis of DivK). PleC exhibits bistability within the crescent-shaped region bounded by the blue lines. (B) One-parameter bifurcation diagrams (signal-response curves) for three different values of <em>k</em><sub>syn_dk</sub>, indicated by the dashed hoizontal lines in panel A. Notice that PleC kinase level is always low if <em>k</em><sub>syn_dk</sub><0.011 and always high if <em>k</em><sub>syn_dk</sub>>0.082.
</strong></p>

<h3>3. The PleC-DivJ-DivK switch confers bistability to the DivL-CckA-CtrA module</h3>

<p align="center"><img "="" src="./CCGraph/26.png" width="60%"></p>
<p class="style1" align="center"> Figure 2.7: <strong>The DivJ-PleC-DivK module controls the DivL-CckA-CtrA module.
When the PleC switch is activated, DivK~P binds to DivL and inactivates components of the CckA module. One parameter bifurcation diagrams show the steady state levels of (A) DivL, (B) CckA kinase, (C) CtrA~P, and (D) CpdR~P as functions of DivJ_tot.
</strong></p>

<div id="Space2"><p style="font-size: 40px">&nbsp;</p></div>
<div id="PleC and DivL Localiztion"><p style="color:rgb(16, 16, 142);font-size: 36;"><b>PleC and DivL Localization</b></p></div>
<div id="Bistable Histidine Model"><p style="font-size: 30px"><b><u>Model</u></b></p></div>
<div id="Space2"><p style="font-size: 16px">&nbsp;</p></div>
<h2>The conundrum of phosphorylation status of PleC and DivK in the predivisional cell</h2>

<p>Progression through the <em>Caulobacter</em> cell cycle is dictated by the phosphorylation status of two proteins, DivK and CtrA. These two proteins are response regulators at the termini of two phosphotransfer modules (DivJ-PleC-DivK and DivL-CckA-CtrA) (Figure 3.1 A). PleC and CckA are bifunctional histidine- modifying enzymes that may act as either kinases or phosphatases [1,2]. Both PleC and DivJ can phosphorylate DivK, while the role of DivL is to up-regulate CckA's kinase activity [3]. (DivL is tyrosine kinase, but its kinase activity is not involved in the up-regulation of CckA [3]; how DivL promotes CckA activity is still unknown.) DivK~P binds to and inhibits DivL [4]. In a swarmer cell, DivJ is absent, PleC is a phosphatase, and DivK is unphosphorylated. Consequently, DivL is actively up-regulating CckA kinase activity [5], which in turn maintains CtrA in its phosphorylated form, thereby inhibiting DNA replication in the swarmer cell [6]. The introduction of DivJ during the swarmer-to-stalked transition enables the phosphorylation of DivK~P, triggering a pathway that culminates in the dephosphorylation of CtrA~P in stalked cells [7-9]. Therefore, at the molecular level, swarmer and stalked cells can be distinguished based on which response regulator-CtrA or DivK is phosphorylated.
</p>

<p>
The schematic in Figure 3.1 A suggests that DivK and CtrA cannot be concurrently phosphorylated during the cell cycle. Therefore, it is surprising that both response regulators are phosphorylated in predivisional cells (Figure 3.1 B). While the level of DivK~P remains fairly constant in the stalked and predivisional stages [10], CtrA~P level changes sharply from lowest in the stalked cell stage to peak activity in the predivisional stage [2,11]. Recent experimental observations [4,5] favor a model where
DivL is active (up-regulating CckA kinase) in the predivisional <em>Caulobacter</em> cell. Hence, DivL, inactivated by DivK~P during the swarmer-to-stalked transition, appears to be reactivated in the predivisional cell, despite the continuing presence of DivK~P. It is unclear how the inhibitory interaction between DivK~P and DivL that is required for swarmer-to-stalked transition early in the cell cycle is circumvented in the predivisional stage of the cell cycle.
</p>

<p align="center"><img "="" src="./CCGraph/31.png" width="70%"></p>
<p class="style1" align="center"> Figure 3.1: <strong>Both DivK and CtrA are phosphorylated during the predivisional stage of the <em>Caulobacter</em> cell cycle. (A) Influence diagram of two signal transduction pathways in C. crescentus. Barbed arrows indicate activation, while the blunt-headed line indicates inhibition. (B) Spatiotemporal distributions of DivK~P (red) and CtrA~P (light blue).
</strong></p>

<h4>Contrasting models of PleC function</h4>

<p>
Of the informal models, the first (in support of PleC kinase) is necessary to explain stalk formation, while the second (in favor of PleC phosphatase) posits conditions that are to be satisfied for replicative asymmetry. While the first model lays out the evolution of PleC function during the course of the cell cycle, the second model only addresses the plausible function of PleC in the predivisional cell. Since PleC-dependent phosphorylation of PleD is required for the development of the stalk, it is fair to assume that PleC is a kinase in the stalked cell. Hence, the difference in the two models can be narrowed down to the suggested function of PleC at the new pole of the predivisional cell (Figure 3.2).
</p>


<p align="center"><img "="" src="./CCGraph/32.png" width="65%"></p>
<p class="style1" align="center"> Figure 3.2: <strong>Two scenarios for the function of PleC (kinase or phosphatase) in the early predivisional (PD) cell. Spatiotemporal dynamics of PleC (green) and DivL (dark blue) during the cell cycle under these two scenarios.
</strong></p>


<p></p>

<h2>Method</h2>


<p>
Our reaction-diffusion model is based on the mechanism proposed in our earlier paper [17]. The proteins under investigation show non-homogeneous distributions along the long axis of a <em>Caulobacter</em> cell; hence, we consider it sufficient to formulate the model for one spatial dimension. The governing partial differential equation (PDE) for a generic chemical species takes the form:
</p>

<p align="center"><img "="" src="./CCGraph/3e1.png" width="35%"></p>

<p>
where C(x,t) is the concentration of species C at location x and time t. The PDEs on which we base our explorations are provided in Table B.1. By discretizing the spatial dimension into n = 100 compartments of equal length h = L/n, where L is the total length of the <em>Caulobacter</em> cell and using a central difference scheme to approximate the Laplacian operator, we convert each PDE into a set of ordinary differential equations (ODEs). In our notation, C<sub>i</sub> is the concentration of species C in compartment i where
</p>

<p align="center"><img "="" src="./CCGraph/3e2.png" width="50%"></p>

<p>
A complete understanding of the mechanism behind localization of DivJ, PleC, DivL and CckA is lacking at this stage, and our model does not attempt to offer one. We enforce the localization of these four kinases based on experimentally observed distributions in wild-type and mutant cells [8,11,12,13,14]. We do this by defining rates of binding and unbinding of species C to docking proteins in compartment i as follows:
</p>



<p align="center"><img "="" src="./CCGraph/3e3.png" width="60%"></p>

<p>
where p<sub>i</sub> is an indicator function that takes the value of 1 or 0, C<sub>i</sub><sup>b</sup> is the concentration of the localized form and C<sub>i</sub><sup>f</sup>  is the concentration of the freely diffusing form of a generic protein in compartment i of the cell.
</p>

<p>
The full set of ODEs were simulated in MATLAB using the ode15s solver [15]. The spatiotemporal distribution plots in the figures were generated using the python library Matplotlib [16]. The colors indicate the concentration gradient from zero (blue) to the maximum value of protein concentration (red) during the cell cycle. A disadvantage of such a plot is that a shallow gradient can be interpreted as significant changes in protein activity and localization. On the other hand, a very steep gradient can result in underestimation of fluctuations in protein activity and localization occurring at the lower range of concentration values. To avoid these problems and to make comparison between wild-type and mutant simulations more convenient, the color bar for each simulation indicates the concentration gradient from zero (blue) to maximum wild-type concentration C<sup>wt_max</sup> (red).
</p>

<div id="Bistable Histidine Model"><p style="font-size: 30px"><b><u>Results</u></b></p></div>
<h3>1. Rapid diffusion of DivK indicates that PleC is a kinase in the predivisional cell. PleC kinase can sequester inhibitor DivK~P from DivL</h3>

<p align="center"><img "="" src="./CCGraph/33.png" width="70%"></p>
<p class="style1" align="center"> Figure 3.3: <strong>Co-localization of PleC kinase and DivL in the early predivisional cell is required for DivL reactivation. Spatiotemporal distributions of proteins during the cell cycle (prior to cytokinesis). Color indicates concentration gradients from minimum (blue) to maximum (red). (A) DivJ is localized at the old pole (t = 30-120 min). The location of PleC is shifted from the old pole (t = 0-50 min) to the new pole of the predivisional cell (t = 90-150 min). Following DivJ localization, the function of PleC changes from a phosphatase to a kinase. (B) Upon phosphorylation, DivK localizes to the poles of the cell. Despite the presence of DivK~P at the new pole of the predivisional cell, DivL is present in the free form (unbound to DivK~P) because DivL co-localizes with PleC kinase and PleC kinase sequesters DivK~P, preventing it from binding to DivL. (C) CckA is uniformly distributed in the swarmer stage and localized at both poles in the predivisional stage. Reactivation of DivL at the new pole results in new-pole CckA becoming a kinase, while old-pole CckA remains a phosphatase. Consequently, the late predivisional cell establishes a gradient of CtrA~P along its length from high at the new pole to low at the old pole.
</strong></p>




<h3>2. The inhibitor-sequestration model reproduces replicative asymmetry in the early predivisional cell and swarmer pole development in the late predivisional cell</h3>

<p align="center"><img "="" src="./CCGraph/34.png" width="65%"></p>
<p class="style1" align="center"> Figure 3.4: <strong>Following cytokinesis, PleC reverts to the phosphatase form.
(A) The total concentration of CckA (blue curve) remains constant during the cell cycle. However, the proportions of phosphatase (red curve) and kinase (green curve) forms of CckA change for each stage of the cell cycle. After cytokinesis (compartmentalization), the concentrations of kinase and phosphatase fraction of CckA in the swarmer and stalked compartments (t = 120-150 min) are similar to their concentration in the non-compartmentalized swarmer (t = 0-30 min) and stalked cell stages (t = 30-90 min) respectively. (B) Spatiotemporal distribution of PleC kinase, DivK~P and CtrA~P after compartmentalization (at t = 120 min). Color indicates concentration gradients from minimum (blue) to maximum (red).
</strong></p>

<h3>3. Loss of asymmetry is a physiological consequence of mutations that alter the spatiotemporal profile of inhibitor sequestration</h3>
<p align="center"><img "="" src="./CCGraph/37.png" width="70%"></p>
<p class="style1" align="center"> Figure 3.5: <strong>PleC kinase conformation is required to establish replicative asymmetry.
Concentration gradients are color coded as in Figure 3.5. (A) In the <em>pleC</em> mutant, free active form of DivL is lower than in wild type, resulting in loss of predivisional cell CtrA~P. (B) In <em>pleC</em><sub>H610A</sub> mutant cells (K<sup>-</sup>P<sup>-</sup>), an elevated level of DivK~P results in less active form of DivL and reduced CtrA~P. (C) In <em>pleC</em><sub>F778L</sub> mutant cells (K<sup>-</sup>P<sup>+</sup>), inhibitor sequestration is retained, resulting in a normal CtrA~P gradient.
</strong></p>



<h3>Proposed molecular mechanism to produce different developmental fates in <em>Caulobacter</em></h3>
<p align="center"><img "="" src="./CCGraph/39.png" width="70%"></p>
<p class="style1" align="center"> Figure 3.6: <strong>Localization and function of the DivK-PleC-DivK and DivL-CckA-CtrA signaling networks, as suggested by the model calculations reported here.
</strong></p>

<p>
	Figure 3.6 summarizes the localization and functional status of proteins, as proposed in our model. At the molecular level, the stalked cell is distinguished from the swarmer cell by the kinase DivJ, which localizes at the old pole and initiates the swarmer-to-stalked transition. Hence, DivJ can be considered as a cell fate determinant protein. However, there is no counterpart for DivJ that initiates the transition from stalked to predivisional cell. Instead, the cell recycles the same components-namely, DivJ-PleC-DivK and DivL-CckA-CtrA to program the predivisional stage of the cell cycle.
</p>

<div id="Space1"><p style="font-size: 12px">&nbsp;</p></div>
<h3><em>Reference</em></h3>
<ol>
	<li>“What Is Mitosis?” Facts, The Public Engagement Team at the Wellcome Genome Campus, 21 July 2021, https://www.yourgenome.org/facts/what-is-mitosis. </li>
	<li>McIntosh, J. Richard, and Michael P. Koonce. “Mitosis.” Science.org, 3 Nov. 1989, https://www.science.org/doi/abs/10.1126/science.2683078. </li>
  <li>Nurse, Paul, et al. “Understanding the Cell Cycle.” Nature Medicine, vol. 4, no. 10, 1998, pp. 1103–1106., https://doi.org/10.1038/2594. </li>

</ol>



<!-- <p>The movies below show the population of PopZ, mRNA and Gene changing during the division process.</p>

<center>

<P>
<EMBED SRC="MultiCurve.avi" autostart="false" width="680" height="500">
</P>

<p class="style1"> Movie 1. PopZ population of four different simulations. If your browser is unable to play this video, right-click over <a href="http://bioinformatics.cs.vt.edu/~cmhshirl/MultiCurve.avi.zip"><em>the link</em></a> and choose save target as.
</p>

<P>
<EMBED SRC="meanCurve.avi" autostart="false" width="680" height="500">
</P>

<p class="style1"> Movie 2. Average population of PopZ, mRNA and Gene (100 simulations). If your browser is unable to play this video, right-click over <a href="http://bioinformatics.cs.vt.edu/~cmhshirl/meanCurve.avi.zip"><em>the link</em></a> and choose save target as.
</p>
 -->


</div> <!-- end #content -->
</td></tr>
</tbody></table>


<div id="FOOTER">
<span style="float:right">Page designed Oct 6, 2014</span><span style="float:left">Site Design by <a class="credits" style="font-weight:800">  © Tyson's Lab</a></span><center><a class="credits" style="font-weight:800" href="mailto:cmhshirl@vt.edu"><img border="0" src="./CCGraph/mailto.gif"> webmaster</a></center>
<div style="clear:both"></div>
</div>
</div>

<script src="drawPleC.js"></script>
<script src="drawPopZ.js"></script>

<script type="text/javascript">


  document.getElementById('project').addEventListener('change', function(){
      if(document.getElementById('project').value =="none"){
        document.getElementById("PleCM").hidden = "hidden";
        document.getElementById("PopZM").hidden = "hidden";
        document.getElementById("g-play-button").hidden = "hidden";
        d3.selectAll("svg").remove();
       }
      else if(document.getElementById('project').value =="PleC"){
        document.getElementById("PleCM").hidden = "";
        document.getElementById("PopZM").hidden = "hidden";
        document.getElementById("g-play-button").hidden = "hidden";
        document.getElementById('project').disabled = true 
      }
      else if(document.getElementById('project').value =="PopZ"){
        document.getElementById("PleCM").hidden = "hidden";
        document.getElementById("PopZM").hidden = "";
        document.getElementById("g-play-button").hidden = "hidden";
        document.getElementById('project').disabled = true      
      }
   });

  function visPleC(){
      var speFlag=[0,0,0,0,0,0,0,0,0,0,0];
      var speNum = 11;
      var species = ["0","1","2","3","4","5","6","7","8","9","10"];
      var label = ["000","101","102","103","104","105","106","107","108","109","110"];

      var j=0;
      var legend = ["none","red","green","blue","cyan","magenta","orange","maroon","lime","black","yellow"];
      for(var i=1; i<speNum; i++){
        if(document.getElementById(species[i]).checked){
          speFlag[i]=1;
          j=j+1
          document.getElementById(label[i]).style.color = legend[j]
          document.getElementById(label[i]).style.fontWeight = "bold"
        }
        else {
          speFlag[i]=0;
          document.getElementById(label[i]).style.color = "grey"
        } 
      document.getElementById(species[i]).disabled = true 
      document.getElementById('submitPleC').disabled = true
      }
      showPleC(speFlag); 
      document.getElementById("g-play-button").hidden = "";
      beginPleC()
  } 

  function visPopZ(){
      var speFlag=[0,0,0,0,0];
      var speNum = 5;
      var species = ["0","21","22","23","24"];
      var label = ["000","201","202","203","204"];

      var j=0;
      var legend = ["none","red","green","blue","cyan","magenta","orange","maroon","lime","black","yellow"];
      for(var i=1; i<speNum; i++){
        if(document.getElementById(species[i]).checked){
          speFlag[i]=1;
          j=j+1
          document.getElementById(label[i]).style.color = legend[j]
          document.getElementById(label[i]).style.fontWeight = "bold"
        }
        else {
          speFlag[i]=0;
          document.getElementById(label[i]).style.color = "grey"
        } 
      document.getElementById(species[i]).disabled = true 
      document.getElementById('submitPopZ').disabled = true
      }
      showPopZ(speFlag); 
      document.getElementById("g-play-button").hidden = "";
      beginPopZ()
  }
</script>




<script type="text/javascript">

/***********************************************
* http://www.dynamicdrive.com/dynamicindex5/dhtmltooltip.htm
***********************************************/

var offsetxpoint=-60 //Customize x offset of tooltip
var offsetypoint=20 //Customize y offset of tooltip
var ie=document.all
var ns6=document.getElementById && !document.all
var enabletip=false
if (ie||ns6)
var tipobj=document.all? document.all["dhtmltooltip"] : document.getElementById? document.getElementById("dhtmltooltip") : ""

function ietruebody(){
return (document.compatMode && document.compatMode!="BackCompat")? document.documentElement : document.body
}

function ddrivetip(thetext, thecolor, thewidth){
if (ns6||ie){
if (typeof thewidth!="undefined") tipobj.style.width=thewidth+"px"
if (typeof thecolor!="undefined" && thecolor!="") tipobj.style.backgroundColor=thecolor
tipobj.innerHTML=thetext
enabletip=true
return false
}
}

function positiontip(e){
if (enabletip){
var curX=(ns6)?e.pageX : event.x+ietruebody().scrollLeft;
var curY=(ns6)?e.pageY : event.y+ietruebody().scrollTop;
//Find out how close the mouse is to the corner of the window
var rightedge=ie&&!window.opera? ietruebody().clientWidth-event.clientX-offsetxpoint : window.innerWidth-e.clientX-offsetxpoint-20
var bottomedge=ie&&!window.opera? ietruebody().clientHeight-event.clientY-offsetypoint : window.innerHeight-e.clientY-offsetypoint-20

var leftedge=(offsetxpoint<0)? offsetxpoint*(-1) : -1000

//if the horizontal distance isn't enough to accomodate the width of the context menu
if (rightedge<tipobj.offsetWidth)
//move the horizontal position of the menu to the left by it's width
tipobj.style.left=ie? ietruebody().scrollLeft+event.clientX-tipobj.offsetWidth+"px" : window.pageXOffset+e.clientX-tipobj.offsetWidth+"px"
else if (curX<leftedge)
tipobj.style.left="5px"
else
//position the horizontal position of the menu where the mouse is positioned
tipobj.style.left=curX+offsetxpoint+"px"

//same concept with the vertical position
if (bottomedge<tipobj.offsetHeight)
tipobj.style.top=ie? ietruebody().scrollTop+event.clientY-tipobj.offsetHeight-offsetypoint+"px" : window.pageYOffset+e.clientY-tipobj.offsetHeight-offsetypoint+"px"
else
tipobj.style.top=curY+offsetypoint+"px"
tipobj.style.visibility="visible"
}
}

function hideddrivetip(){
if (ns6||ie){
enabletip=false
tipobj.style.visibility="hidden"
tipobj.style.left="-1000px"
tipobj.style.backgroundColor=''
tipobj.style.width=''
}
}

document.onmousemove=positiontip

</script>


</body></html>

